k8s/sandbox/papi-common/papi/namespace.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+   name: papi-sbox
```

having a namespace after a cluster is weird is it not multiple teams using it?
anyway, the env shouldn't be in it.
```suggestion
   name: papi
```